test/integration/strategies/layers/connector/base/BaseConnectorTest.t.sol

```diff
@@ -118,7 +118,7 @@ abstract contract BaseConnectorTest is PRBTest, StdUtils, StdCheats {
 
   function testFork_deposit_RevertWhen_InvalidToken() public {
     address token = address(1);
-    vm.expectRevert(abi.encodeWithSelector(BaseConnector.InvalidDepositToken.selector, token));
+    vm.expectRevert();
```

Comment:
Given our change in the code, we will try to perform the `transferFrom` before actually reverting. While it would be best to fail with a better error, it's easier (and saves space) to write it this way